README.md

```diff
@@ -12,6 +12,13 @@ Create `indexer/.dev.vars` file with the following content:
 ```
 SECRET_HEADER_KEY=X-SECRET-KEY
 SECRET_HEADER_VALUE=SecretToken
+CHAINALYSIS_API_KEY=YourChainalysisApiKey
+```
+
+Create `terminator/.dev.vars` file with the following content:
+
+```
+FILCDN_CONTROLLER_ADDRESS_PRIVATE_KEY=0xSomePrivateKey
 ```
```

Comment:
It is key to have these files present when running `npm run build:types`, as that's the way how `wrangler` discovers what secrets are available in the environment.

We often forget to create these files. When that happens the next time (after my PR was landed), we will get TypeScript compiler errors flagging that secrets are not available on the `Env` object. That could be confusing, as it's not clear how to fix it.

How can we improve our DX here? I can imagine creating a script to populate `.dev.vars` files with some sensible defaults, WDYT? But then we need to remember to update that script whenever we add a new secret 🤔